grib2_to_cb/test/test_int_metar_model_grib.py

```diff
@@ -77,9 +77,12 @@ def test_grib_builder_one_thread_file_pattern_hrrr_ops_conus():
         # 1632423600  September 23, 2021 19:00:00 2126617000001
         # first_epoch = 1634252400 - 10
         # last_epoch = 1634252400 + 10
-        credentials_file = os.environ["HOME"] + "/adb-cb1-credentials"
+        credentials_file = os.environ["CREDENTIALS"]
         # remove output files
-        for _f in glob("/opt/data/grib2_to_cb/output/test1/*.json"):
+        if not os.path.exists('/opt/data/grib2_to_cb/hrrr_ops/output/test1'):
+            os.makedirs('/opt/data/grib2_to_cb/hrrr_ops/output/test1')
```

Comment:
Instead of using a hard-coded path for the test output, would it be possible to pytest's `tmp_path` fixture? It will handle creating the directory as well as cleaning up after itself.

https://docs.pytest.org/en/7.1.x/how-to/tmp_path.html

We'd be able to use it in this test like so:

```python
def test_grib_builder_one_thread_file_pattern_hrrr_ops_conus(tmp_path):

# Remove the code that cleans up the previous test...

vx_ingest.runit(
  {
    # The non-path variables stay the same...
    "output_dir": f"{tmp_path}",
  }
)

for _f in glob(f"{tmp_path}/*.json"):
  ...
```

Reply:
OK, I don't see any problem with this and it does clean up nicer.